### ISSUE TITLE
Why is my nuxt app color mode configuration not working with the "prefrence : light" ? 

### ISSUE THREAD

**Issue by amirsafaricg** [nuxt-modules/color-mode#262]
this is my nuxt color mode config in nuxt.config.ts: 

` colorMode: {
    classSuffix: '',
    preference: 'light',
    fallback: 'light',
  },`
  
  its starting always by default dark mode. how Can i fix it and put the prefrence on system ?

**Comment by NenoSun** [comment]
I am also having the same issue.

**Comment by akncnkoc** [comment]
i am also having the same issue

**Comment by MuhammadM1998** [comment]
TLDR: Try removing color-mode cookie and it should work fine

Just faced this issue and the problem that there was already a color mode cookie with `value=system` (the default). It looks like the module only creates the cookie if there's none while it should always override it or override when the cookie value != `preference`.

**Comment by damisparks** [comment]
It is related to #263
## Recommendation
Ideally, you'd like to dark mode by default, Here is what your config should look like.
The solution works for me.
```ts
export default defineNuxtConfig({
  colorMode: {
    preference: 'dark',
    fallback: 'dark'
  }
})
```
### Note 💡 
If you're stuck in light mode even after changing this setting, you might need to remove the `nuxt-color-mode` entry from your browser's local storage.

**Comment by Liuzhch1** [comment]
Remove the `nuxt-color-mode` in local storage works! Thanks a lot

**Comment by karladler** [comment]
> Note 💡
> If you're stuck in light mode even after changing this setting, you might need to remove the nuxt-color-mode entry from your browser's local storage.

I think this is a bug that should be resolved.

**Comment by hacknug** [comment]
> > Note 💡
> > If you're stuck in light mode even after changing this setting, you might need to remove the nuxt-color-mode entry from your browser's local storage.
> 
> I think this is a bug that should be resolved.

It's not a bug if it is respecting your stored settings. Might be a good idea to add some notes to the docs but that is as far as I'd take it.

**Comment by ArnarValur** [comment]
Just a funny comment, maybe somebody is overlooking what I've been tearing my hair off for few moments now.
I use a Firefox Developer Browser, and for some ungodly reason the browser has Website Appearance set on "Light mode", and not "Automatic". I facepalmed my head on the desk. (I've been troubleshooting why the hell Light mode is always on even tho the app Im developing has system chosen, and my OS is set on Dark mode. :')

**Comment by lpknv** [comment]
> TLDR: Try removing color-mode cookie and it should work fine
> 
> Just faced this issue and the problem that there was already a color mode cookie with `value=system` (the default). It looks like the module only creates the cookie if there's none while it should always override it or override when the cookie value != `preference`.

That is not applicable for the causal end user in my case. I cannot expect a normal non-IT person to clear cookies and browser cache.
